articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -76,6 +80,35 @@ az ml data create -f [PATH_TO_YML_FILE] --workspace-name [YOUR_AZURE_WORKSPACE]
 [!Notebook-python[] (~/azureml-examples-main/sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb?name=upload-data)]
 ---
 
+If you already have your data present in Azure Blob Storage and want to create data asset out of it, you can do so by providing path to the location in Azure Blob Storage as shown below.
```

Comment:
I think we should also content here on how to link an existing "Azure blob storage" into a user's AML workspace. The current changes assume that the blob storage is already registered as a datastore in AML workspace.